src/ec/packages/ec-component-description-list/description-list.test.js

```diff
@@ -46,5 +46,14 @@ describe('EC - Description list', () => {
 
       return expect(render(withExtraAttributes)).resolves.toMatchSnapshot();
     });
+
+    test('with missing input data and debug enabled shows the right warning message', () => {
+      expect.assertions(1);
+
+      const dataCompliance = { ...data, _compliance_: true };
+      dataCompliance.items[0].term = '';
```

Comment:
Lack of definition item case not covered.

Reply:
Ok, i improved the test, as you noticed the idea was more to test the "feature" than covering as many as the use cases we have, thanks for pushing on this ;)